modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/RatedRequest.java

```diff
@@ -48,32 +49,64 @@
  * */
 @SuppressWarnings("unchecked")
 public class RatedRequest extends ToXContentToBytes implements Writeable {
-    private String specId;
-    private SearchSourceBuilder testRequest;
-    private List<String> indices = new ArrayList<>();
-    private List<String> types = new ArrayList<>();
+    private String id;
+    private List<String> indices;
```

Comment:
Since we cannot make sure these are set, can we make indices and types empty lists by default?